### PR TITLE
Added API Call:retrieve all members of departments

### LINKED ISSUE #84: Server: Retrieve all members of a department
Retrieve the id and the name of the members of a specific department.

### PR DESCRIPTION
**Pull request by KevinGus**
Added api route /deparment-members in membersRoutes.
Added the functionality of retrieveDepartmentMembers in membersController.
Closes #84

### CODE REVIEW

**Review by christophtrybek — APPROVED** [review]
Nice Job. Looks Good To Me.

**Review by LennartRukower — CHANGES_REQUESTED** [review]
Very good first pr!

**Review by christophtrybek — APPROVED** [review]
LGTM

File: server/src/members/membersController.ts [hunk 1]
```diff
@@ -121,6 +121,27 @@ export const retrieveMember = (req: Request, res: Response): void => {
   }
 };
 
+/**
+ * Retrieves all members of a department
+ */
+export const retrieveDepartmentMembers = (req: Request, res: Response): void => {
+  database.query(
+    `SELECT mitgliedID, vorname, nachname, ressort, bezeichnung
```
Comment (LennartRukower) [inline server/src/members/membersController.ts:129]: Delete trailing whitespace

File: server/src/members/membersController.ts [hunk 2]
```diff
@@ -121,6 +121,27 @@ export const retrieveMember = (req: Request, res: Response): void => {
...
 };
 
+/**
+ * Retrieves all members of a department
+ */
+export const retrieveDepartmentMembers = (req: Request, res: Response): void => {
+  database.query(
+    `SELECT mitgliedID, vorname, nachname, ressort, bezeichnung 
+    FROM mitglied, ressort
+    WHERE ressort = ressortID AND mitgliedstatus <= 3
```
Comment (LennartRukower) [inline server/src/members/membersController.ts:131]: Delete trailing whitespace

**Review by LennartRukower — CHANGES_REQUESTED** [review]
Please always resolve problems, thrown by TSLint (like trailing whitespaces).
Do you have TSLint activated?

**Review by LennartRukower — APPROVED** [review]

**Pull request merged**